src/server/test/web/readingsBarGroupQuantity.js

```diff
@@ -122,87 +127,113 @@ mocha.describe('readings API', () => {
                     const expected = await parseExpectedCsv('src/server/test/web/readingsData/expected_bar_group_ri_15-20_mu_kWh_gu_kWh_st_-inf_et_inf_bd_76.csv');
                     // Create a request to the API for unbounded reading times and save the response
                     const res = await chai.request(app).get(`/api/unitReadings/bar/groups/${GROUP_ID}`)
-                        .query({ 
-                            timeInterval: ETERNITY.toString(), 
+                        .query({
+                            timeInterval: ETERNITY.toString(),
                             barWidthDays: '76',
-                            graphicUnitId: unitId });
+                            graphicUnitId: unitId
+                        });
                     // Check that the API reading is equal to what it is expected to equal
                     expectReadingToEqualExpected(res, expected, GROUP_ID);
                 });
-                // Add BG7 here
+                
+             
+                    mocha.it("BG7: 13 day bars for 15 + 20 minute reading intervals and quantity units with reduced, partial days & kWh as kWh", async () => {
+                        // Define arrays of data for units, conversions, and test meters/groups
+                        const unitData = unitDatakWh;
```

Comment:
I'm not sure it is best to reassign the name of the standard units & conversions to use them. The other test cases have not, as far as I know, done this. For these reasons, I would directly use the name and remove these two lines.